test/functional/framework.go

```diff
@@ -282,10 +299,15 @@ func (f *FluentdFunctionalFramework) DeployWithVisitors(visitors []runtime.PodBu
 func (f *FluentdFunctionalFramework) addOutputContainers(b *runtime.PodBuilder, outputs []logging.OutputSpec) error {
 	log.V(2).Info("Adding outputs", "outputs", outputs)
 	for _, output := range outputs {
-		if output.Type == logging.OutputTypeFluentdForward {
+		switch output.Type {
+		case logging.OutputTypeFluentdForward:
 			if err := f.AddForwardOutput(b, output); err != nil {
 				return err
 			}
+			//		case logging.OutputTypeSyslog:
```

Comment:
Remove comment?

Reply:
sorry i messed up. this should not  be commented. I was testing another BZ (sending over syslog, but no syslog receiver running), so i commented this.

Reply:
there were some `FIt` focussed tests also, have removed them now